targets/unity-v2/make.js

```diff
@@ -484,7 +484,9 @@ function getCustomApiLogic(tabbing, apiCall)
 {
     if (apiCall.name === "ExecuteFunction")
         return tabbing + "var localApiServerString = PlayFabSettings.LocalApiServer;\n" +
-            tabbing + "if (!string.IsNullOrEmpty(localApiServerString))\n" +
+            tabbing + "var debugFunctions = PlayFabSettings.LocalDebugFunctions;\n" + 
+            tabbing + "var shouldDebug = debugFunctions.Contains(\"*\") || debugFunctions.Contains(request.FunctionName);\n" +
```

Comment:
Equals rather than Contains for the first clause. Plus see comments on other PRs about whether we should do the specific functions thing.

Reply:
This comment applies to all the PRs. Just realized my disconnect here. Contains works, although I'd prefer it if we required "*" to be the only thing in the list. 

And I'm still not convinced we should do the specific functions, especially for private preview.

Reply:
If it's done and it works why would we remove it?

Reply:
@MGudgin waiting for your response before taking action on this.